packages/plugins/src/cli/setup.ts

```diff
@@ -137,31 +137,23 @@ export const makeCLI = (
       'install',
       'Installs the required version of the Sphinx Solidity library contracts and Sphinx Foundry fork',
       (y) =>
-        y
-          .usage('Usage: sphinx install')
-          .option('skip-library', {
-            describe:
-              'Skips installing the Sphinx library and only installs the Sphinx Foundry fork.',
-            boolean: true,
-            default: false,
-          })
-          .option('ci', {
-            describe:
-              'Continuous integration mode (skips installing the Sphinx Foundry fork)',
-            boolean: true,
-            default: false,
-          }),
+        y.usage('Usage: sphinx install').option('skip-library', {
+          describe:
+            'Skips installing the Sphinx library and only installs the Sphinx Foundry fork.',
+          boolean: true,
+          default: false,
+        }),
       async (argv) => {
-        const { skipLibrary, ci } = argv
+        const { skipLibrary } = argv
 
-        if (skipLibrary && ci) {
+        if (skipLibrary) {
```

Comment:
Seems like the "skip library" flag isn't necessary? Also, if the user enters `--skip-library`, this if-statement will always throw an error